### PR TITLE
Fixes #2 : Provisional Board Management UI

### LINKED ISSUE #2: Build a board management UI
The idea is to build a board management UI where one user (admin) can create several chess boards in which other users can play against each other or the admin user itself can play with himself.

Building this UI is agnostic of the chess board itself since building the chess board is a separate task. Also this UI will be agnostic of backend and how the users will actually connect to the platform. 

The focus here is on the layout, functionality and user friendliness.

Different scenarios to consider:
- Admin user should be able to create multiple boards
- Admin user should be able to decide who plays on each board
- All users can play against one or more games simultaneously
...etc

### PR DESCRIPTION
**Pull request by Smit2010**
@statebait Please review and suggest changes

### CODE REVIEW

File: src/components/dashboard/Dashboard.tsx [hunk 1]
```diff
@@ -0,0 +1,164 @@
+import React, { useState } from "react";
```
Comment (statebait) [inline src/components/dashboard/Dashboard.tsx]: Rename this to index.tsx so the imports will be /components/dashboard

Breakdown this component into smaller components.

File: src/components/dashboard/components/OpenGames.tsx [hunk 2]
```diff
@@ -0,0 +1,218 @@
+import React from "react";
```
Comment (statebait) [inline src/components/dashboard/components/OpenGames.tsx]: Breakdown the component into smaller components.

File: src/components/dashboard/components/NewGameDialog.tsx [hunk 3]
```diff
@@ -0,0 +1,468 @@
+import React, { useState } from "react";
```
Comment (statebait) [inline src/components/dashboard/components/NewGameDialog.tsx]: Breakdown this component.

**Review by statebait — CHANGES_REQUESTED** [review]
Some changes to be done:

Avoid using inline styling. Use emotion for styling. eg:

Create a file called styles.ts for a specific component:

```jsx
import { css } from "emotion";

export const someStyle = css`
    color: red;
`
```

In your component:

```jsx
import { someStyle } from "./styles.ts";
....
<Component className={someStyle}>
....
</Component>
```

Make sure to change this everywhere.

Please remove the package-lock.json. We are using yarn as our package manager.

**Comment by Smit2010** [comment]
@statebait Please review and suggest any changes if any. Thanks.

**Review by statebait — APPROVED** [review]
LGTM!

**Comment by statebait** [comment]
give @Smit2010 150 points

**Pull request merged**